.github/CODEOWNERS

```diff
@@ -25,30 +25,30 @@
 
 /sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @prmathur-microsoft @bikamani @barustum      %DigitalTwins
 
-/sdk/eventgrid/                                 @Kishp01 @ahamad-MS   
+/sdk/eventgrid/                                 @Kishp01 @ahamad-MS   %Event Grid
```

Comment:
As we chatted we might want to encode the labels as a comment. We can certainly try the % but I worry some folks parsing this file might break.

Reply:
We talked a bit more about this and the `#` in the same line might also be treated in an undefined way.

We are going to go ahead with the current approach and we will roll it back if that causes problems.

We might consider trying it out in the `azure-sdk-tools` repo first as that has lower churn and lower traffic.

/cc @danieljurek